tests/test-cases/profiling/data.csv

```diff
@@ -0,0 +1,6 @@
+Dimension,Value,Measure,Unit
```

Comment:
What's this file for?

Reply:
I am not sure. I don't remember creating the file. I can delete it of course

Reply:
sorted